tests/plotting/test_plot_manager.py

```diff
@@ -76,6 +77,10 @@ class PlotRefreshTester:
 
     def __init__(self, root_path: Path):
         self.plot_manager = PlotManager(root_path, self.refresh_callback)
+        # Set a very high refresh interval here to avoid unintentional refresh cycles
+        self.plot_manager.refresh_parameter.interval_seconds = 10000
+        # Set to the current time to avoid automated refresh after we start below.
+        self.plot_manager.last_refresh_time = time.time()
         self.plot_manager.start_refreshing()
```

Comment:
why not move this into `run()` instead? it could then run `stop_refreshing()` before returning. To isolate the tests and not have refreshes happen outside of them

Reply:
or we could construct a plot manager every time we run a test.

Reply:
I intentionally moved it out of `run()` in https://github.com/Chia-Network/chia-blockchain/pull/9588. If you in that branch run the test without 24b2121 you will see a failure. We missed the failure of the crashing plot manager thread because we just restarted the thread in `run()` if it crashed. We could probably have a separate test for that also though.